modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -61,6 +61,30 @@ export async function verifyEdShareProof(
   return decodedUValueProof === rawUValueProof;
 }
 
+/**
+ * Verify an Ecdsa KeyShare Proof.
+ *
+ * @param senderPubKey public key of the sender of the privateShareProof
+ * @param privateShareProof u value proof
+ * @param uValue u value from an Ecdsa keyshare
+ * @return {boolean} whether uValue proof actually was signed by sender as part of their subkeys
+ */
+export async function verifyEcShareProof(
```

Comment:
since majority of the verify is the same 
we can do with `verifyShareProof(..., algo)`?

Reply:
good point